.github/workflows/ci-tests-data.yml

```diff
@@ -94,7 +94,12 @@ jobs:
 
       - name: Testing Data
         working-directory: tests/tests_data
+        env:
```

Comment:
Note that this means no developer out of lightning org can contribute

Reply:
The bucket is public
Just need to be auth to AWS
Any dev can contribute if they use their own credentials

Reply:
removed.